### PR TITLE
Release v0.8.0

### LINKED ISSUE #1459: RELEASE: v0.8.0
**NOTE**: Don't forget to limit the pod name for the update script to 63 chars.

https://github.com/kinvolk/lokomotive/blob/6db76b4ba247188e30cdae79081d4de989f4155e/scripts/update/0.6.1-0.7.0/update.sh#L15

### PR DESCRIPTION
**Pull request by surajssd**
Fixes #1459

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,123 @@
+## v0.8.0 - 2021-05-25
+
+We're happy to announce the release of Lokomotive v0.8.0 (TODO).
+
+### Changes in v0.8.0
+
+#### Kubernetes updates
+
+- Update AKS to `1.18.17` ([#1466](https://github.com/kinvolk/lokomotive/pull/1466)).
+#### Component updates
```
Comment (iaguis) [inline CHANGELOG.md:10]: Maybe a new line above this?

**Comment by surajssd** [comment]
I will squash the first two commits, once I know that it is okay to run multiple `kubectl apply` over downloading a tar.

File: docs/how-to-guides/certificate-rotation.md [hunk 2]
```diff
@@ -43,6 +43,14 @@ The date in the `notAfter` line is the expiration date of the cluster CA certifi
 
 ## Step 2: Rotate certificates
 
+Lokoctl transfers the newly generated certificates to the controller nodes over SSH. Run the
+following command to load the ssh key:
```
Comment (iaguis) [inline docs/how-to-guides/certificate-rotation.md:48]:
```suggestion
Lokoctl transfers the newly generated certificates to the controller nodes over SSH. Run the
following command to load the one of the SSH keys specified in the cluster configuration (in this case `~/.ssh/id_rsa`):
```

Comment (iaguis) [reply docs/how-to-guides/certificate-rotation.md:48]: Maybe also add a note mentioning that on Equinix Metal you need to have the public IP from the machine where you run lokoctl added to the `management_cidrs` argument in the cluster configuration (I ran into this issue when testing :D).

**Review by iaguis — CHANGES_REQUESTED** [review]
Some small things.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,130 @@
...
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/hostendpoints.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ipamblocks.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ipamconfigs.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ipamhandles.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ippools.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/kubecontrollersconfigurations.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/networkpolicies.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/networksets.yaml
```
Comment (knrt10) [inline CHANGELOG.md:62]: This does not look good to look at honestly

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,130 @@
...
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.46.0/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.46.0/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.46.0/example/prometheus-operator-crd/monitoring.coreos.com_probes.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.46.0/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.46.0/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.46.0/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
```
Comment (knrt10) [inline CHANGELOG.md:119]: Same here, does not look good. Maybe we can add it in a gist and apply that single file.

**Review by knrt10 — COMMENTED** [review]
Some comments

File: CHANGELOG.md (hunk 3 above)
Comment (surajssd) [reply CHANGELOG.md:62]: I agree this is not pretty.

File: docs/how-to-guides/certificate-rotation.md (hunk 2 above)
Comment (surajssd) [reply docs/how-to-guides/certificate-rotation.md]: Done, PTAL.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,118 @@
+## v0.8.0 - 2021-05-25
+
+We're happy to announce the release of Lokomotive v0.8.0 (TODO).
```
Comment (knrt10) [inline CHANGELOG.md:3]: I think this can also be changed now?

Comment (surajssd) [reply CHANGELOG.md:3]: Oh, good. Thanks for catching that :laughing:

Comment (surajssd) [reply CHANGELOG.md:3]: All reviews are addressed.

Comment (surajssd) [reply CHANGELOG.md]: Done.

**Review by knrt10 — APPROVED** [review]
I did not test it, but docs LGTM +1

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,130 @@
...
+#### Features
+
+- Add a certificate rotation command: `lokoctl cluster certificate rotate` ([#1435](https://github.com/kinvolk/lokomotive/pull/1435)).
+- Add `reclaim_policy` field to components `rook-ceph`, `openebs-storage-class` and `aws-ebs-csi-driver`. Change the default behaviour of the default storage class to `Retain` from `Delete`. ([#1369](https://github.com/kinvolk/lokomotive/pull/1369)).
+
+#### Deprecation and Removal
+
+- Remove the `webhook` field from the `cert-manager` component ([#1413](https://github.com/kinvolk/lokomotive/pull/1413)).
+
+#### Configuration syntax changes
```
Comment (iaguis) [inline CHANGELOG.md:30]: This is usually under `Updating from vX.X.X`.

Comment (surajssd) [reply CHANGELOG.md:32]: Done.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,130 @@
+## v0.8.0 - 2021-05-25
```
Comment (iaguis) [inline CHANGELOG.md:1]:
```suggestion
## v0.8.0 - 2021-05-26
```

**Review by iaguis — APPROVED** [review]
lgtm

**Pull request merged**